tdp_vars_defaults/livy/livy.yml

```diff
@@ -1,8 +1,11 @@
 ---
 # Livy version
-livy_release: apache-livy-0.7.1-incubating-bin
+livy_release: apache-livy-0.8.0-incubating-TDP-0.1.0-SNAPSHOT-bin
 livy_dist_file: "{{ livy_release }}.zip"
 
+# Spark version ('spark2' or 'spark3')
```

Comment:
I use this variable to name the systemd service: `livy-server-spark2.service` or `livy-server-spark3.service`

Reply:
Inside `livy.yml` we can specify `spark3` for the version and it will work ?

Reply:
It will work but it won't make sense indeed. You mean I should remove the comment? `# Spark version ('spark2' or 'spark3')`

Reply:
Otherwise I can add 2 variables for the service:
```yaml
livy_server_service_name: livy-server-spark2
livy_server_service_desc: Livy Server for Spark 2
```

Reply:
Yes the comment say that we can specify `spark2` or `spark3` for the version in the `livy.yml` but if you can specify "spark3" for livy that means you can use livy to launch spark3 code but I feel like livy-spark3 is needed to work with spark3. Or maybe I didn't understand and the livy service can launch spark3 code and the livy-spark3 service is just in case we want 2 livy server at the same time.

If, as a user, I just want to launch spark3 code with livy, I can use livy service with `spark3` version or I must use the livy-spark3 service ?

Reply:
LGTM, not need to change the comment.

Reply:
The only problem is how to differentiate the Ansible groups targeted by the 2 different installs if we go for a single service?